src/software/world/field.h

```diff
@@ -144,6 +144,13 @@ class Field
      */
     Point centerPoint() const;
 
+    /**
+     * Returns the center line of the field (as a segment)
```

Comment:
Just to be extra clear, can we specify in the comment that the center line runs between the center of each goal? (so as to not get confused with the halfway line)

Edit: JK even I got confused. I think naming this the `halfwayLine` would be more clear.